MIGRATE-DEV.md

```diff
@@ -11,6 +11,7 @@ Version 1.16 (Unreleased)
 - `\humhub\modules\content\components\ContentAddonActiveRecord::canWrite()`
 - `\humhub\modules\file\models\File::canRead()` use `canView()` instead
 - `\humhub\modules\content\components\ContentAddonActiveRecord::canRead()` use `canView()` instead
+- `\humhub\models\Setting::isInstalled()` use `BaseSettingsManager::isInstalled()` instead
```

Comment:
I like the improvement. Just as a thought, could this also be something for `ApplicationTrait`?  

e.g. `Yii::$app->isInstalled()`. 


```php
    public static function isInstalled(): bool
    {
        return isset($this->params['installed']) && $this->params['installed'];
    }

```

Reply:
Yes, I guess that would be an even better solution! (Although the method would then have to be non-static)

If we do this, I'd like to also move the `BaseSettingsManager::isDatabaseInstalled()` there. And maybe even the `DynamicSetting:set(Database)Installed()`, too.

Would you agree?

Reply:
Hmmm, good point.

Yes, then let's move all 4 methods `is|set[Database]Installed()` to `ApplicationTrait`. Seems to be the best place.

Reply:
on it!